beacon-chain/core/epoch/epoch_operations.go

```diff
@@ -310,7 +310,7 @@ func TotalAttestingBalance(
 // Spec pseudocode definition:
 //    epochs_since_finality = (state.slot - state.finalized_slot) // EPOCH_LENGTH
```

Comment:
Can you please update the pseudocode definitions across as well?

Reply:
done !